index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Lato"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="css/main.css" />
+    <title>Microverse Concert</title>
+  </head>
+  <body>
+    <div>
+      <nav class="navigation">
+        <section class="top-bar">
+          <i class="fa fa-facebook"></i>
+          <i class="fa fa-instagram"></i>
+          <i class="fa fa-youtube"></i>
+          <a href="#">Login</a>
+        </section>
+        <section class="flex-nav">
+          <div class="menu-open">
+            <img src="assets/img/ic_menu.svg" alt="" />
+          </div>
+          <div class="menu">
+            <a href="index.html" class="logo"
+              ><img src="assets/img/logo.png" alt=""
+            /></a>
+            <a href="about.html">About</a>
+            <a href="#media">Media</a>
+            <a href="#lineup">Line up</a>
+            <a href="#buytickets" id="buy">Buy tickets</a>
+            <div class="menu-close">
+              <img src="assets/img/Union.svg" alt="" />
+            </div>
+          </div>
+        </section>
+      </nav>
+    </div>
```

Comment:
[OPTIONAL] I strongly recommend wrapping this section in a `header` tag, this would improve your HTML semantics.

Reply:
As using semantic tags is part of the requirement of this project, it will be better to make this comment required.